recipes/pdielec/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "PDielec" %}
+{% set version = "6.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 57c12cb29768dd9b49c7c17e12bf6570fa56a135a824f394a7538f011255f577
+
+build:
+  number: 0
```

Comment:
Also, as it appears you don't have any platform- or python-version-specific dependencies: i'd recommend  adding `noarch: python`: it will substantially reduce your CI build burden (1 build vs 9+)

Reply:
Yes, I will apply this.  Although there are only python modules loaded, there are dependencies on the operating system through mesa or windows or mac and the way they interact with PyQt and PyOpenGl.  I will remove the yum installation requests and see if that will still work

Reply:
Still needed yum installation requests so added the file back in